assets/js/theme.js

```diff
@@ -104,6 +104,12 @@ const toggleTheme = () => {
   }
 };
 
+document.addEventListener("keydown", function (e) {
+  if ((e.ctrlKey || e.metaKey) && e.shiftKey && e.keyCode === 76) {
```

Comment:
is `ctrl/command + shift + l` generally the default keybinding for dark mode? I am not sure

Reply:
lol it's `cmd + shift + l` in notion, so I assumed it would be like this in other apps. I'm not sure.